Client/Frontend/Browser/TabTrayButtonExtensions.swift

```diff
@@ -13,6 +13,9 @@ class PrivateModeButton: ToggleButton, PrivateModeUI {
         accessibilityHint = .TabTrayToggleAccessibilityHint
         let maskImage = UIImage(named: ImageIdentifiers.privateMaskSmall)?.withRenderingMode(.alwaysTemplate)
         setImage(maskImage, for: [])
+        showsLargeContentViewer = true
+        largeContentTitle = .TabTrayToggleAccessibilityLabel
+        largeContentImage = maskImage
```

Comment:
I reused an existing accessibility label.  Should I create a separate localized string for the `largeContentTitle`?